acceptancetests/assess_bundle_export.py

```diff
@@ -1,77 +0,0 @@
-#!/usr/bin/env python
```

Comment:
We probably don't actually want to remove the test, but just not have it hooked up to a jenkins job right?

Reply:
Yes. we don't have to. Just to keep it clean in this branch.
Also, I closed the other PR which i created in qa_jenkins. But that is required for 2.5 ci test as we have removed the dev-mode flag there.
If we have this test here it can trigger and the command is not exposed. So better to clean.

Reply:
@howbazaar - Do i make sense here. correct me if Iam wrong please.

Reply:
I think it is fine to keep it. In particular the feature flag around bundle-export should be removed again once we release 2.4.2.

It is much easier to revert a very small change, and there is less churn on the codebase. Can we please keep it in the tree?